mapper-processor/src/main/java/com/datastax/oss/driver/internal/mapper/processor/util/generation/GeneratedCodePatterns.java

```diff
@@ -222,8 +245,23 @@ public static void setValue(
           .addStatement("$T $L = $L.newValue()", UdtValue.class, udtValueName, udtTypeName);
       String childHelper = enclosingClass.addEntityHelperField(entityClass);
       methodBuilder
-          .addStatement("$L.set($L, $L)", childHelper, valueName, udtValueName)
+          // driver doesn't have the ability to send partial UDT, unset values values will be
+          // serialized to null - set NullSavingStrategy.DO_NOT_SET explicitly
+          .addStatement(
+              "$L.set($L, $L,  $T.$L)",
+              childHelper,
+              valueName,
+              udtValueName,
+              NullSavingStrategy.class,
+              NullSavingStrategy.DO_NOT_SET)
           .addStatement("$1L = $1L.setUdtValue($2L, $3L)", targetName, cqlName, udtValueName)
+          .endControlFlow()
+          .beginControlFlow(
```

Comment:
Note that
```java
          .endControlFlow()
          .beginControlFlow(
```
Can be shortened as:
```java
          .nextControlFlow(
```
I'm addressing this directly during the merge.